Magento_Directory/templates/currency.phtml

```diff
@@ -19,15 +19,15 @@
     <ul class="dropdown-list__list dropdown-list__item--parent">
         <li class="dropdown-list__item">
             <span class="dropdown-list__link"
-                  data-dropdown="language-<?php echo $block->escapeHtml($block->getCurrentCurrencyCode()) ?>"
+                  aria-controls="language-<?php echo $block->escapeHtml($block->getCurrentCurrencyCode()) ?>"
             >
                 <?php echo __('Currency') ?>: <?php echo $block->escapeHtml($currentCurrencyCode) ?> - <?php echo @$block->escapeHtml($currencies[$currentCurrencyCode]) ?>
                 <svg class="dropdown-list__icon dropdown-list__icon--inner">
```

Comment:
plaese add title for svg and role attr like here https://github.com/SnowdogApps/magento2-alpaca-components/tree/master/components/01-globals/icons